packages/block-library/src/accordion-item/index.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/**
+ * Server-side rendering of the `core/accordion-item` block.
+ *
+ * @package WordPress
+ * @since 6.6.0
+ *
+ * @param array $attributes The block attributes.
+ * @param string $content The block content.
+ *
+ * @return string Returns the updated markup.
+ */
+function block_core_accordion_item_render( $attributes, $content ) {
+	if ( ! $content ) {
+		return $content;
+	}
+
+	$p         = new WP_HTML_Tag_Processor( $content );
+	$unique_id = wp_unique_id( 'accordion-item-' );
+
+	// If the item is open by default,
+	// add its id to the accordion group's initial context.isOpen list.
+	if ( $attributes['openByDefault'] ) {
+		// phpcs:ignore Gutenberg.CodeAnalysis.ForbiddenFunctionsAndClasses.ForbiddenFunctionCall
+		gutenberg_block_core_accordion_group_item_ids( $unique_id );
+	}
+
+	// Initialize the state of the item on the server using a closure,
+	// since we need to get derived state based on the current context.
+	wp_interactivity_state(
+		'core/accordion',
+		array(
+			'isOpen' => function () {
+				$context = wp_interactivity_get_context();
+				return in_array( $context['id'], $context['isOpen'], true );
+			},
+		)
+	);
+
+	if ( $p->next_tag( array( 'class_name' => 'wp-block-accordion-item' ) ) ) {
+		$p->set_attribute( 'data-wp-context', '{ "id": "' . $unique_id . '" }' );
+		$p->set_attribute( 'data-wp-class--is-open', 'state.isOpen' );
+
+		if ( $p->next_tag( array( 'class_name' => 'accordion-item__toggle' ) ) ) {
+			$p->set_attribute( 'data-wp-on--click', 'actions.toggle' );
+			$p->set_attribute( 'aria-controls', $unique_id );
+			$p->set_attribute( 'data-wp-bind--aria-expanded', 'state.isOpen' );
+
+			if ( $p->next_tag( array( 'class_name' => 'wp-block-accordion-content' ) ) ) {
+				$p->set_attribute( 'aria-labelledby', $unique_id );
```

Comment:
I believe `aria-labelledby` should reference an `id` attribute, so we may need to set an `id` alongside the `aria-controls` on line 47, e.g.

```
$p->set_attribute( 'id', $unique_id );
```

Reply:
Good catch, fixed here: [88428a6](https://github.com/WordPress/gutenberg/pull/64119/commits/88428a676911e577cdf5bc420ddd1bbd52f7bbf2)